Dockerfiles/midstream/unit_tests.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
```

Comment:
I would include file into container image.

That way you needn;t build anything because you have all necessary dependencies already in midstream image including `run_tests.py` which you need to test.

And for running tests you will just use podman -ti --rm run  $name_of_midstream_image python3 unit_tests.py 

It will simplify the unit test itself as well. You will be able to get rid of all podman login.

Reply:
Unit tests are now part of container image.